web/src/features/filters/components/filter-builder.tsx

```diff
@@ -368,11 +368,20 @@ function FilterBuilderForm({
                         <SelectValue placeholder="" />
                       </SelectTrigger>
                       <SelectContent>
-                        {["true", "false"].map((option) => (
-                          <SelectItem key={option} value={option}>
-                            {option}
-                          </SelectItem>
-                        ))}
+                        {filter.column === "Maintainer"
```

Comment:
can you use true / false or string options here? The filterbuilder should be abstract and now we move column specific logic in here.

Happy to support when you are blocked here.